shared/app.js

```diff
@@ -70,6 +73,8 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
+      var ClientRouter = this.options.ClientRouter || require(defaultRouterModule);
```

Comment:
Hmm, that doesn't seem to fix it for me, i'm getting an error on this line https://github.com/rendrjs/rendr/blob/master/shared/base/router.js#L184 - it looks like `realAction` is a string not the action that is expected.

In the meantime i'm going to revert the commit that caused the breakage and push out `1.0.4` (i think it's that version?) so it's in a working state for everyone.